R/glmnot.R

```diff
@@ -121,14 +121,27 @@ glmnot.default <- function(X, y, family,
                            pmax = +Inf,
                            formula = NULL,
                            xlev = NULL,
-                           weights = rep(1, nrow(X))) {
-  m <- cv.glmnet(X, y, family = family,
-              alpha = alpha,
-              type.measure = type.measure,
-              pmax = pmax,
-              nfolds = nfolds,
-              weights = weights,
-              parallel = TRUE)
+                           weights = rep(1, nrow(X)),
+                           intercept = TRUE
+                           ) {
+  if (pmax == +Inf)
```

Comment:
Could you make this a [do.call](https://www.rdocumentation.org/packages/base/versions/3.5.3/topics/do.call) to avoid the duplication? i.e. conditionally add `pmax` to the arglist if it's supplied